TESTS/mbedmicro-rtos-mbed/basic/main.cpp

```diff
@@ -44,7 +44,7 @@ void print_tick_thread() {
 }
 
 int main() {
-    GREENTEA_SETUP(total_ticks + 5, "timing_drift_auto");
+    GREENTEA_SETUP(total_ticks + 5, "default_auto");
```

Comment:
This is an incorrect change, the host test should be set to `timing_drift_auto`. Please revert this.